diff --git a/bin/varnishtest/gensequences b/bin/varnishtest/gensequences
index 4337186b8..18a68b43b 100644
--- a/bin/varnishtest/gensequences
+++ b/bin/varnishtest/gensequences
@@ -70,7 +70,7 @@ while (getline > 0) {
 		l_prefix_parent[n] = prefix;
 		l_prefix_suffix[n] = sequence[i];
 		if (!l_prefix_name[n])
-			l_prefix_name[n] = "teken_state_" ++npr;
+			l_prefix_name[n] = "teken_state_" "" ++npr;
 		prefix = n;
 	}
 
